### ISSUE TITLE
TakeoverNotificationViewController crash line 87

### ISSUE THREAD

**Issue by abrown252** [mixpanel/mixpanel-swift#22]
Crash when TakeoverNotificationViewController displayed, line 87: ```window.isHidden = false``` throws ```SIGABRT```, seems it can't find the nib file. Installed via cocoapods using. Full stack trace:

```
2016-09-27 11:11:09.515611 LuckyTrip[1629:311525] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Could not load NIB in bundle: 'NSBundle </var/containers/Bundle/Application/4CB1773C-4169-4081-9CD6-6BAC6296B515/LuckyTrip.app> (loaded)' with name 'TakeoverNotificationViewController~iphoneportrait''
*** First throw call stack:
(0x18d64c1c0 0x18c08455c 0x18d64c108 0x19399683c 0x1937f7648 0x1935c9868 0x193495074 0x193494fa4 0x19349b750 0x193498bf0 0x101121268 0x1010f1f10 0x1010f291c 0x1010f19e0 0x1019a925c 0x1019a921c 0x1019ae284 0x18d5f9f2c 0x18d5f7b18 0x18d526048 0x18efa9198 0x193500628 0x1934fb360 0x10007a148 0x18c5085b8)
libc++abi.dylib: terminating with uncaught exception of type NSException
```
<img width="1066" alt="screen shot 2016-09-27 at 11 12 18" src="https://cloud.githubusercontent.com/assets/662520/18869229/4e5cb56c-84a3-11e6-9a7a-ebe550c0babe.png">

**Comment by yarneo** [comment]
hey @abrown252 ! Thanks for letting us know, we will look into this straight away and update you on our findings as soon as possible

**Comment by yarneo** [comment]
We found the cause and will send out a new release today. thanks for letting us know!

**Comment by yarneo** [comment]
This was fixed in the latest release v2.0.1! Let us know if that resolved your issue. Thanks!

**Comment by abrown252** [comment]
hey @yarneo that worked perfectly, thanks.
